### PR TITLE
[ ADF-3821] Start standalone task e2e tests

### PR DESCRIPTION
**Pull request by rgherghelas**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e/process-services-cloud/start_task_custom_app_cloud.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        tasksCloudDemoPage.createNewTask();
+        startTask.addDueDate('12/12/2018')
+                 .checkStartButtonIsEnabled();
+        startTask.addDueDate('invalid date')
+                 .blur(startTask.dueDate)
+                 .validateDate(dateValidationError)
+                 .checkStartButtonIsDisabled();
+    });
+
+    xit('[C290182] Should be possible to assign the task to another user (ADF-3828 issue)', () => {
```
Comment (eromano) [inline e2e/process-services-cloud/start_task_custom_app_cloud.e2e.ts]: can we move this test in a separate PR?

File: e2e/process-services-cloud/start_task_custom_app_cloud.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+describe('Start Task', () => {
+
+    const settingsPage = new SettingsPage();
+    const loginSSOPage = new LoginSSOPage();
+    const navigationBarPage = new NavigationBarPage();
+    const appListCloudComponent = new AppListCloudComponent();
+    const tasksCloudDemoPage = new TasksCloudDemoPage();
+    const startTask = new StartTasksCloudComponent();
+    const standaloneTaskName = Util.generateRandomString(5);
+    const taskNameLessThen255Characters = Util.generateRandomString(100);
```
Comment (eromano) [inline e2e/process-services-cloud/start_task_custom_app_cloud.e2e.ts]: can we use 255 and 256?

File: e2e/pages/adf/demo-shell/tasksCloudDemoPage.ts [hunk 3]
```diff
@@ -51,4 +53,29 @@ export class TasksCloudDemoPage {
         Util.waitUntilElementIsVisible(this.activeFilter);
         return this.activeFilter.getText();
     }
+
+    createNewTask() {
```
Comment (cristinaj) [inline e2e/pages/adf/demo-shell/tasksCloudDemoPage.ts]: The name of this method makes me think this method is creating a task. Maybe you could rename it to something that indicates that the method is just clicking on two buttons in order to open the start task dialog.

File: e2e/process-services-cloud/start_task_custom_app_cloud.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+    it('[C290166] Should be possible to cancel a task', () => {
+        tasksCloudDemoPage.createNewTask();
+        startTask.checkStartButtonIsDisabled()
+                 .blur(startTask.name)
+                 .checkValidationErrorIsDisplayed(requiredError);
+        startTask.addName(standaloneTaskName)
+                 .addDescription('descriptions')
+                 .addDueDate('12/12/2018')
+                 .clickStartButton();
```
Comment (cristinaj) [inline e2e/process-services-cloud/start_task_custom_app_cloud.e2e.ts]: I'm not sure if it's needed to click on start button in this test case

File: e2e/process-services-cloud/start_task_custom_app_cloud.e2e.ts [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        startTask.addName(taskName255Characters)
+                 .checkStartButtonIsEnabled();
+        startTask.addName(taskNameBiggerThen255Characters)
+                 .blur(startTask.name)
+                 .checkValidationErrorIsDisplayed(lengthValidationError)
+                 .checkStartButtonIsDisabled()
+                 .clickCancelButton();
+    });
+
+    it('[C290181] Should be displayed an error message if the date is invalid', () => {
```
Comment (cristinaj) [inline e2e/process-services-cloud/start_task_custom_app_cloud.e2e.ts]: The testrail ID is the same with the one above.
